src/sentry/static/sentry/app/components/smartSearchBar/index.tsx

```diff
@@ -69,7 +82,10 @@ const getInputButtonStyles = p => `
     getMediaQuery(theme.breakpoints[p.collapseIntoEllipsisMenu], 'none')};
 `;
 
-const getDropdownElementStyles = p => `
+const getDropdownElementStyles = (p: {
+  showBelowMediaQuery: keyof Theme['breakpoints'][number];
```

Comment:
oops, this should just be number like `collapseIntoEllipsisMenu`, couldn't get the keyof stuff working